bazel/external/jinja.BUILD

```diff
@@ -1,8 +1,60 @@
 licenses(["notice"])  # Apache 2
 
+# Jinja2 2.11 sources are stored in 'src/jinja2' compared to 2.10's 'jinja2'.
+# We cannot make a py_library naively with glob, because then the result package would be
+# 'src.jinja2', instead of 'jinja2'.
+# Cannot work around this with stuff like `import src.jinja2` because it looks like jinja2 calls
+# its own packages via reflection.
+# So solving it the trivial way, moving the source files up one directory.
+# If http_archive's 'strip_prefix' attribute could strip multiple segments instead of just one,
+# we could have just ignored 'jinja-2.x.y/src' instead of doing all this trickery here.
+
+filegroup(
+    name = "jinja2_sources",
+    srcs = glob(["src/jinja2/*.py"]),
+)
+
+genrule(
+    name = "move_sources_up_one_level",
+    srcs = ["//:jinja2_sources"],
+    outs = [
+        "jinja2/__init__.py",
+        "jinja2/_compat.py",
+        "jinja2/_identifier.py",
+        "jinja2/asyncfilters.py",
+        "jinja2/asyncsupport.py",
+        "jinja2/bccache.py",
+        "jinja2/compiler.py",
+        "jinja2/constants.py",
+        "jinja2/debug.py",
+        "jinja2/defaults.py",
+        "jinja2/environment.py",
+        "jinja2/exceptions.py",
+        "jinja2/ext.py",
+        "jinja2/filters.py",
+        "jinja2/idtracking.py",
+        "jinja2/lexer.py",
+        "jinja2/loaders.py",
+        "jinja2/meta.py",
+        "jinja2/nativetypes.py",
+        "jinja2/nodes.py",
+        "jinja2/optimizer.py",
+        "jinja2/parser.py",
+        "jinja2/runtime.py",
+        "jinja2/sandbox.py",
+        "jinja2/tests.py",
+        "jinja2/utils.py",
+        "jinja2/visitor.py",
+    ],
+    cmd = """
+        touch $(@D)/jinja2 && \
+        for f in $(SRCS); do cp $$f $(@D)/jinja2/$$(basename $$f); done
+    """,
+)
+
 py_library(
     name = "jinja2",
-    srcs = glob(["jinja2/**/*.py"]),
+    srcs = ["//:move_sources_up_one_level"],
```

Comment:
Could you use https://docs.bazel.build/versions/master/be/python.html#py_binary.imports to enable the glob to still work?